CHANGELOG.md

```diff
@@ -5,6 +5,23 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
+## Release 3.11.0 (2020-05-06)
+
+[Changes since v3.10.0](https://github.com/realm/realm-studio/compare/v3.10.0...v3.11.0)
+
+### Enhancements
+
+- Added support for reading / writing Realm files using Realm file format v10. When opening a file which is using an older format, you will get prompted to upgrade (and optionally backup) the Realm file. ([#1278](https://github.com/realm/realm-studio/pull/1278))
+
```

Comment:
Wondering if we should actually list which versions of the SDKs this is required for?

Reply:
It would be JS v6, Java v7,  Cocoa v5, .NET v5.0

Reply:
I added a suggestion above.